scripts/sct_run_batch.py

```diff
@@ -0,0 +1,155 @@
+
+import spinalcordtoolbox.image as msct_image
+from spinalcordtoolbox.image import Image, concat_data
+from spinalcordtoolbox.utils import Metavar, SmartFormatter
+
+parser = argparse.ArgumentParser(
+  description = 'Wrapper to processing scripts, which loops across subjects. Data should be '
+                'organized according to the BIDS structure: '
+                'https://github.com/sct-pipeline/spine_generic#file-structure.\n'
+                'The processing script (task) should accept a subject directory as its only argument. '
+                'Additional information is passed via environment variables \n'
+                'PATH_SEGMANUAL (--path-segmanual)\n'
+                'PATH_DATA (--data-path)\n'
+                'PATH_RESULTS (--out-path)\n'
+                'PATH_LOG (--out-path with log appended)\n'
+                'PATH_QC (--out-path with QC appended)\n'
+                'ITK_GLOBAL_DEFAULT_NUMBER_OF_THREADS (--itk-threads)\n',
+  formatter_class = SmartFormatter,
+  prog=os.path.basename(__file__).strip('.py'))
+
+parser.add_argument("--jobs", type = int, default = 1,
+                    help = 'The number of jobs to run in parallel. '
+                           'either an integer greater than or equal to one '
+                           'specifying the number of cores, 0 or a negative integer '
+                           'specifying number of cores minus that number. For example '
+                           '--jobs -1 indicates run ncores - 1 jobs in parallel.')
+parser.add_argument("--data-path", help = 'Path containing subject directories in the BIDS format')
+parser.add_argument("--subject-prefix", default = "sub-",
+                    help = 'Subject prefix, defaults to "sub-"')
+parser.add_argument("--out-path", default = "./",
+                    help = 'Output path, subdirectories for results, logs, and QC '
+                           'will be generated here')
+parser.add_argument("--include",
+                    help = 'Optional regex used to filter the list of subject directories. Only process '
+                           'a subject if they match the regex. Inclusions are processed before exclusions')
+parser.add_argument("--exclude",
+                    help = 'Optional regex used to filter the list of subject directories. Only process '
+                           'a subject if they do not match the regex. Exclusions are processed '
+                           'after inclusions')
+parser.add_argument("--path-segmanual", default = ".",
+                    help = 'A path containing manual segmentations to be used by the task program.')                
+parser.add_argument("--itk-threads", default = 1,
+                    help = 'Number of threads to use for ITK based programs including ANTs. Set to a low '
+                           'number to avoid a large increase in memory. Defaults to 1')
+parser.add_argument("task",
+                    help = 'Shell script used to process the data.'
```

Comment:
```suggestion
                    help = 'Shell script used to process the data.')
```

Reply:
This is fixed in my new batch of commits